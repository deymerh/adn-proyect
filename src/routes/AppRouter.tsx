import React from 'react';
import { BrowserRouter, Switch, Route } from 'react-router-dom';
import ContainerDonaciones from '../pages/IngresoDonaciones/index';
import MostrarDonaciones from '../pages/MostrarDonaciones/index';

const AppRouter: React.FC = () => {
  return (
    <BrowserRouter>
      <div>
        <Switch>
          <Route path="/" exact component={ContainerDonaciones} />
          <Route path="/donaciones" exact component={MostrarDonaciones} />
        </Switch>
      </div>
    </BrowserRouter>
  );
};

export default AppRouter;
