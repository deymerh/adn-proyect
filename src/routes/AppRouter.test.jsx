import { render } from '@testing-library/react';
import AppRouter from './AppRouter';

describe('Pruebas en el <AppRouter/>', () => {
  test('Crear el SnapShot del compoente', () => {
    const page = render(<AppRouter/>)
    expect(page).toMatchSnapshot();
  });
  
});
