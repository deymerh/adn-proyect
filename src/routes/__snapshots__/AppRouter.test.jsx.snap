// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en el <AppRouter/> Crear el SnapShot del compoente 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div>
        <div
          class="MuiGrid-root makeStyles-container-1 MuiGrid-container"
        >
          <h5
            class="MuiTypography-root MuiTypography-h5 MuiTypography-colorTextSecondary"
          >
            Ingresar donación
          </h5>
          <div
            class="MuiGrid-root MuiGrid-container MuiGrid-item MuiGrid-justify-content-xs-center MuiGrid-grid-xs-10 MuiGrid-grid-md-6"
          >
            <div
              class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-md-6"
            >
              <div
                class="MuiFormControl-root MuiTextField-root makeStyles-field-5 MuiFormControl-fullWidth"
              >
                <label
                  class="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-shrink MuiInputLabel-marginDense MuiInputLabel-outlined MuiFormLabel-filled Mui-required Mui-required"
                  data-shrink="true"
                >
                  Nombre
                  <span
                    aria-hidden="true"
                    class="MuiFormLabel-asterisk MuiInputLabel-asterisk"
                  >
                     
                    *
                  </span>
                </label>
                <div
                  class="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-fullWidth MuiInputBase-formControl MuiInputBase-marginDense MuiOutlinedInput-marginDense"
                >
                  <input
                    aria-invalid="false"
                    class="MuiInputBase-input MuiOutlinedInput-input MuiInputBase-inputMarginDense MuiOutlinedInput-inputMarginDense"
                    name="nombre"
                    required=""
                    type="text"
                    value="Deymer"
                  />
                  <fieldset
                    aria-hidden="true"
                    class="PrivateNotchedOutline-root-7 MuiOutlinedInput-notchedOutline"
                  >
                    <legend
                      class="PrivateNotchedOutline-legendLabelled-9 PrivateNotchedOutline-legendNotched-10"
                    >
                      <span>
                        Nombre
                         *
                      </span>
                    </legend>
                  </fieldset>
                </div>
              </div>
              <div
                class="MuiFormControl-root MuiTextField-root makeStyles-field-5 MuiFormControl-fullWidth"
              >
                <label
                  class="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-shrink MuiInputLabel-marginDense MuiInputLabel-outlined MuiFormLabel-filled Mui-required Mui-required"
                  data-shrink="true"
                >
                  Id
                  <span
                    aria-hidden="true"
                    class="MuiFormLabel-asterisk MuiInputLabel-asterisk"
                  >
                     
                    *
                  </span>
                </label>
                <div
                  class="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-fullWidth MuiInputBase-formControl MuiInputBase-marginDense MuiOutlinedInput-marginDense"
                >
                  <input
                    aria-invalid="false"
                    class="MuiInputBase-input MuiOutlinedInput-input MuiInputBase-inputMarginDense MuiOutlinedInput-inputMarginDense"
                    name="id"
                    required=""
                    type="number"
                    value="73984410"
                  />
                  <fieldset
                    aria-hidden="true"
                    class="PrivateNotchedOutline-root-7 MuiOutlinedInput-notchedOutline"
                  >
                    <legend
                      class="PrivateNotchedOutline-legendLabelled-9 PrivateNotchedOutline-legendNotched-10"
                    >
                      <span>
                        Id
                         *
                      </span>
                    </legend>
                  </fieldset>
                </div>
              </div>
              <div
                class="MuiFormControl-root MuiTextField-root makeStyles-field-5 MuiFormControl-fullWidth"
              >
                <label
                  class="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-shrink MuiInputLabel-marginDense MuiInputLabel-outlined MuiFormLabel-filled Mui-required Mui-required"
                  data-shrink="true"
                >
                  CantidadDonada
                  <span
                    aria-hidden="true"
                    class="MuiFormLabel-asterisk MuiInputLabel-asterisk"
                  >
                     
                    *
                  </span>
                </label>
                <div
                  class="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-fullWidth MuiInputBase-formControl MuiInputBase-marginDense MuiOutlinedInput-marginDense"
                >
                  <input
                    aria-invalid="false"
                    class="MuiInputBase-input MuiOutlinedInput-input MuiInputBase-inputMarginDense MuiOutlinedInput-inputMarginDense"
                    name="cantidadDonada"
                    required=""
                    type="number"
                    value="10000"
                  />
                  <fieldset
                    aria-hidden="true"
                    class="PrivateNotchedOutline-root-7 MuiOutlinedInput-notchedOutline"
                  >
                    <legend
                      class="PrivateNotchedOutline-legendLabelled-9 PrivateNotchedOutline-legendNotched-10"
                    >
                      <span>
                        CantidadDonada
                         *
                      </span>
                    </legend>
                  </fieldset>
                </div>
              </div>
              <div
                class="MuiFormControl-root MuiTextField-root makeStyles-field-5 MuiFormControl-fullWidth"
              >
                <label
                  class="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-shrink MuiInputLabel-marginDense MuiInputLabel-outlined MuiFormLabel-filled Mui-required Mui-required"
                  data-shrink="true"
                >
                  Edad
                  <span
                    aria-hidden="true"
                    class="MuiFormLabel-asterisk MuiInputLabel-asterisk"
                  >
                     
                    *
                  </span>
                </label>
                <div
                  class="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-fullWidth MuiInputBase-formControl MuiInputBase-marginDense MuiOutlinedInput-marginDense"
                >
                  <input
                    aria-invalid="false"
                    class="MuiInputBase-input MuiOutlinedInput-input MuiInputBase-inputMarginDense MuiOutlinedInput-inputMarginDense"
                    name="edad"
                    required=""
                    type="number"
                    value="17"
                  />
                  <fieldset
                    aria-hidden="true"
                    class="PrivateNotchedOutline-root-7 MuiOutlinedInput-notchedOutline"
                  >
                    <legend
                      class="PrivateNotchedOutline-legendLabelled-9 PrivateNotchedOutline-legendNotched-10"
                    >
                      <span>
                        Edad
                         *
                      </span>
                    </legend>
                  </fieldset>
                </div>
              </div>
              <div
                class="MuiFormControl-root MuiTextField-root makeStyles-field-5 MuiFormControl-fullWidth"
              >
                <label
                  class="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-shrink MuiInputLabel-marginDense MuiInputLabel-outlined MuiFormLabel-filled Mui-required Mui-required"
                  data-shrink="true"
                >
                  Fecha
                  <span
                    aria-hidden="true"
                    class="MuiFormLabel-asterisk MuiInputLabel-asterisk"
                  >
                     
                    *
                  </span>
                </label>
                <div
                  class="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-fullWidth MuiInputBase-formControl MuiInputBase-marginDense MuiOutlinedInput-marginDense"
                >
                  <input
                    aria-invalid="false"
                    class="MuiInputBase-input MuiOutlinedInput-input MuiInputBase-inputMarginDense MuiOutlinedInput-inputMarginDense"
                    name="fecha"
                    required=""
                    type="date"
                    value="2021-07-30"
                  />
                  <fieldset
                    aria-hidden="true"
                    class="PrivateNotchedOutline-root-7 MuiOutlinedInput-notchedOutline"
                  >
                    <legend
                      class="PrivateNotchedOutline-legendLabelled-9 PrivateNotchedOutline-legendNotched-10"
                    >
                      <span>
                        Fecha
                         *
                      </span>
                    </legend>
                  </fieldset>
                </div>
              </div>
              <button
                class="MuiButtonBase-root MuiButton-root MuiButton-contained makeStyles-btn-3 MuiButton-containedPrimary MuiButton-containedSizeSmall MuiButton-sizeSmall MuiButton-fullWidth"
                tabindex="0"
                type="button"
              >
                <span
                  class="MuiButton-label"
                >
                  Guardar
                </span>
                <span
                  class="MuiTouchRipple-root"
                />
              </button>
            </div>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div>
      <div
        class="MuiGrid-root makeStyles-container-1 MuiGrid-container"
      >
        <h5
          class="MuiTypography-root MuiTypography-h5 MuiTypography-colorTextSecondary"
        >
          Ingresar donación
        </h5>
        <div
          class="MuiGrid-root MuiGrid-container MuiGrid-item MuiGrid-justify-content-xs-center MuiGrid-grid-xs-10 MuiGrid-grid-md-6"
        >
          <div
            class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-md-6"
          >
            <div
              class="MuiFormControl-root MuiTextField-root makeStyles-field-5 MuiFormControl-fullWidth"
            >
              <label
                class="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-shrink MuiInputLabel-marginDense MuiInputLabel-outlined MuiFormLabel-filled Mui-required Mui-required"
                data-shrink="true"
              >
                Nombre
                <span
                  aria-hidden="true"
                  class="MuiFormLabel-asterisk MuiInputLabel-asterisk"
                >
                   
                  *
                </span>
              </label>
              <div
                class="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-fullWidth MuiInputBase-formControl MuiInputBase-marginDense MuiOutlinedInput-marginDense"
              >
                <input
                  aria-invalid="false"
                  class="MuiInputBase-input MuiOutlinedInput-input MuiInputBase-inputMarginDense MuiOutlinedInput-inputMarginDense"
                  name="nombre"
                  required=""
                  type="text"
                  value="Deymer"
                />
                <fieldset
                  aria-hidden="true"
                  class="PrivateNotchedOutline-root-7 MuiOutlinedInput-notchedOutline"
                >
                  <legend
                    class="PrivateNotchedOutline-legendLabelled-9 PrivateNotchedOutline-legendNotched-10"
                  >
                    <span>
                      Nombre
                       *
                    </span>
                  </legend>
                </fieldset>
              </div>
            </div>
            <div
              class="MuiFormControl-root MuiTextField-root makeStyles-field-5 MuiFormControl-fullWidth"
            >
              <label
                class="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-shrink MuiInputLabel-marginDense MuiInputLabel-outlined MuiFormLabel-filled Mui-required Mui-required"
                data-shrink="true"
              >
                Id
                <span
                  aria-hidden="true"
                  class="MuiFormLabel-asterisk MuiInputLabel-asterisk"
                >
                   
                  *
                </span>
              </label>
              <div
                class="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-fullWidth MuiInputBase-formControl MuiInputBase-marginDense MuiOutlinedInput-marginDense"
              >
                <input
                  aria-invalid="false"
                  class="MuiInputBase-input MuiOutlinedInput-input MuiInputBase-inputMarginDense MuiOutlinedInput-inputMarginDense"
                  name="id"
                  required=""
                  type="number"
                  value="73984410"
                />
                <fieldset
                  aria-hidden="true"
                  class="PrivateNotchedOutline-root-7 MuiOutlinedInput-notchedOutline"
                >
                  <legend
                    class="PrivateNotchedOutline-legendLabelled-9 PrivateNotchedOutline-legendNotched-10"
                  >
                    <span>
                      Id
                       *
                    </span>
                  </legend>
                </fieldset>
              </div>
            </div>
            <div
              class="MuiFormControl-root MuiTextField-root makeStyles-field-5 MuiFormControl-fullWidth"
            >
              <label
                class="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-shrink MuiInputLabel-marginDense MuiInputLabel-outlined MuiFormLabel-filled Mui-required Mui-required"
                data-shrink="true"
              >
                CantidadDonada
                <span
                  aria-hidden="true"
                  class="MuiFormLabel-asterisk MuiInputLabel-asterisk"
                >
                   
                  *
                </span>
              </label>
              <div
                class="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-fullWidth MuiInputBase-formControl MuiInputBase-marginDense MuiOutlinedInput-marginDense"
              >
                <input
                  aria-invalid="false"
                  class="MuiInputBase-input MuiOutlinedInput-input MuiInputBase-inputMarginDense MuiOutlinedInput-inputMarginDense"
                  name="cantidadDonada"
                  required=""
                  type="number"
                  value="10000"
                />
                <fieldset
                  aria-hidden="true"
                  class="PrivateNotchedOutline-root-7 MuiOutlinedInput-notchedOutline"
                >
                  <legend
                    class="PrivateNotchedOutline-legendLabelled-9 PrivateNotchedOutline-legendNotched-10"
                  >
                    <span>
                      CantidadDonada
                       *
                    </span>
                  </legend>
                </fieldset>
              </div>
            </div>
            <div
              class="MuiFormControl-root MuiTextField-root makeStyles-field-5 MuiFormControl-fullWidth"
            >
              <label
                class="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-shrink MuiInputLabel-marginDense MuiInputLabel-outlined MuiFormLabel-filled Mui-required Mui-required"
                data-shrink="true"
              >
                Edad
                <span
                  aria-hidden="true"
                  class="MuiFormLabel-asterisk MuiInputLabel-asterisk"
                >
                   
                  *
                </span>
              </label>
              <div
                class="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-fullWidth MuiInputBase-formControl MuiInputBase-marginDense MuiOutlinedInput-marginDense"
              >
                <input
                  aria-invalid="false"
                  class="MuiInputBase-input MuiOutlinedInput-input MuiInputBase-inputMarginDense MuiOutlinedInput-inputMarginDense"
                  name="edad"
                  required=""
                  type="number"
                  value="17"
                />
                <fieldset
                  aria-hidden="true"
                  class="PrivateNotchedOutline-root-7 MuiOutlinedInput-notchedOutline"
                >
                  <legend
                    class="PrivateNotchedOutline-legendLabelled-9 PrivateNotchedOutline-legendNotched-10"
                  >
                    <span>
                      Edad
                       *
                    </span>
                  </legend>
                </fieldset>
              </div>
            </div>
            <div
              class="MuiFormControl-root MuiTextField-root makeStyles-field-5 MuiFormControl-fullWidth"
            >
              <label
                class="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-shrink MuiInputLabel-marginDense MuiInputLabel-outlined MuiFormLabel-filled Mui-required Mui-required"
                data-shrink="true"
              >
                Fecha
                <span
                  aria-hidden="true"
                  class="MuiFormLabel-asterisk MuiInputLabel-asterisk"
                >
                   
                  *
                </span>
              </label>
              <div
                class="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-fullWidth MuiInputBase-formControl MuiInputBase-marginDense MuiOutlinedInput-marginDense"
              >
                <input
                  aria-invalid="false"
                  class="MuiInputBase-input MuiOutlinedInput-input MuiInputBase-inputMarginDense MuiOutlinedInput-inputMarginDense"
                  name="fecha"
                  required=""
                  type="date"
                  value="2021-07-30"
                />
                <fieldset
                  aria-hidden="true"
                  class="PrivateNotchedOutline-root-7 MuiOutlinedInput-notchedOutline"
                >
                  <legend
                    class="PrivateNotchedOutline-legendLabelled-9 PrivateNotchedOutline-legendNotched-10"
                  >
                    <span>
                      Fecha
                       *
                    </span>
                  </legend>
                </fieldset>
              </div>
            </div>
            <button
              class="MuiButtonBase-root MuiButton-root MuiButton-contained makeStyles-btn-3 MuiButton-containedPrimary MuiButton-containedSizeSmall MuiButton-sizeSmall MuiButton-fullWidth"
              tabindex="0"
              type="button"
            >
              <span
                class="MuiButton-label"
              >
                Guardar
              </span>
              <span
                class="MuiTouchRipple-root"
              />
            </button>
          </div>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
